Containerfile.helpers-ci

```diff
@@ -0,0 +1,11 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-builder-multi-openshift-4.18 AS builder
```

Comment:
question: is this something we are going to manually update every release?

Reply:
For now, probably. The good news is that for the CI use-case, the CI system should keep that up-to-date for us and we're not depending on anything else being in the image (aside from `oc`, which the CI system will inject for us anyway).

Reply:
Another solution is that we could replace these with `fedora:latest` or whatever since all we really need is a Golang toolchain. I only chose the OCP builder image because I knew it has everything we need.